### PR TITLE
Visually distinguish docs on new site

### PR DESCRIPTION
**Pull request by etefera**
Adds a dropdown on every page with a link to the old site.
Addresses https://github.com/GoogleContainerTools/kpt/issues/1662 for new site.
Preview at https://kpt-dev-ee2a9--version-dropdown-98kk6v2g.web.app

### CODE REVIEW

**Comment by frankfarzan** [comment]
1. Let's do the overview page changes separately. There's a larger story for that: https://github.com/GoogleContainerTools/kpt/issues/1681
2. The version drop down makes more sense in the navbar.
3. We should use a specific version instead of `next`. This will be tied to the specific version linked in the installation page.

**Comment by etefera** [comment]
> 1. Let's do the overview page changes separately. There's a larger story for that: #1681
> 2. The version drop down makes more sense in the navbar.
> 3. We should use a specific version instead of `next`. This will be tied to the specific version linked in the installation page.

Addressed.

**Comment by frankfarzan** [comment]
1. Rendering is broken on mobile
[Screen Shot 2021-05-20 at 5 38 03 PM
](url)

2. `Version 0.39` -> `v0.39` `v1.0.0-alpha.3`

File: site/static/js/plugins.js [hunk 1]
```diff
@@ -43,6 +43,23 @@ function convertFromHugo(content) {
   return content.replace(hugoHideDirectives, "").replace(hugoDirectiveTags, "");
 }
 
+async function addVersionDropdown() {
+  const sidebar = document.getElementsByClassName("sidebar").item(0);
+  const versionDropdown = `
+  <div class="dropdown">
+                <button class="btn btn-primary dropdown-toggle" type="button" data-toggle="dropdown">Version: 1.0.0-alpha.3
```
Comment (frankfarzan) [inline site/static/js/plugins.js:50]: There needs to be a fast follow to properly version the docs in the CD pipeline. There are not multiple pages that need to be manually updated whenever there's a release.

**Review by frankfarzan — APPROVED** [review]

**Pull request merged**